app/views/search/index.html.haml

```diff
@@ -40,10 +40,13 @@
         .search-page-options{class:  @search_results.empty? ? 'justify-content-end': ''}
           - unless @search_results.empty?
             .search-page-number-of-results
-              = "#{t('search.match_in')} #{@search_results.length} #{t('search.ontologies')} #{t('federation.from')} #{request_portals_names.join(', ').html_safe} (#{sprintf("%.2f", @time)}s)".html_safe
+              = "#{t('search.match_in')} #{@search_results.length} #{t('search.ontologies')} #{t('federation.from')} #{request_portals_names(@federation_counts).join(', ').html_safe} in #{sprintf("%.2f", @time)}s".html_safe
```

Comment:
here we need to extract the common code with browse page

Reply:
It's not the same sentence the only extractable thing is the name of the portals + the count which is already extracted in request_portals_names function

Reply:
done in (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/778/commits/0559a3d9596d7b0239fb0017fa1ae3c2e59e99b4)